drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -24,6 +24,10 @@
 #define SHIM_CLKCTL_LPGPDMA_SPA	BIT(0)
 #define SHIM_CLKCTL_LPGPDMA_CPA	BIT(8)
 
+# define DSP_INIT_LPGPDMA(x)	  (0x71A60 + (2*x))
```

Comment:
This is another spot where we're still using hard-coded addresses and not a dts device.  Also: is the address really the same betwen cAVS and ACE?  In general everything moved around with ACE, even if the devices were the same hardware.

Reply:
it is the same. I agree about the DTS, but I need to figure out the right place to put it. This register is exactly part of the gpdma but part of resource control. But yeah, these things will properly come along in DTS.